Makefile

```diff
@@ -1,5 +1,5 @@
 # goreleaser removes the `v` prefix when building and this does too
-VERSION = 0.5.0
+VERSION = 0.6.1
```

Comment:
The `setup-new-aws-user version` command successfully prints out the version, but it was printing out the wrong version. I think we forgot to bump this during prior changes. This brings things up to date for a patch version release.

Lemme know if this should actually be a major or minor version bump. The error code was `2` for a panic, and now it's `1` for an error. That's _technically_ a change in behavior, but probably not a super significant one.

Reply:
I think an exit code of  1 is better in this situation